README.md

```diff
@@ -77,9 +77,11 @@ Item encoding | Metadata encoding | Pattern
 `ValueCoding` | `NSCoding`        | ValueWithObjectMetadata
 `ValueCoding` | `ValueCoding`     | ValueWithValueMetadata
 
-There are also two styles of API. The *functional* API works on `YapDatabase` types, `YapDatabaseReadTransaction`, `YapDatabaseReadWriteTransaction` and `YapDatabaseConnection`. The *persistable* API works on your `Persistable` types directly, and receives the `YapDatabase` type as arguments.
+## Swifty APIs
```

Comment:
"Extension APIs" sounds good; I'll update my branch when I get back to my computer. Thanks!

> On Jan 6, 2016, at 17:33, Daniel Thorpe <notifications@github.com> wrote:
> 
> In README.md:
> 
> > @@ -77,9 +77,11 @@ Item encoding | Metadata encoding | Pattern
> >  `ValueCoding` | `NSCoding`        | ValueWithObjectMetadata
> >  `ValueCoding` | `ValueCoding`     | ValueWithValueMetadata
> >  
> > -There are also two styles of API. The *functional* API works on `YapDatabase` types, `YapDatabaseReadTransaction`, `YapDatabaseReadWriteTransaction` and `YapDatabaseConnection`. The *persistable* API works on your `Persistable` types directly, and receives the `YapDatabase` type as arguments.
> > +## Swifty APIs
> How about "Extension APIs"?
> 
> —
> Reply to this email directly or view it on GitHub.
>